NineChronicles.Headless/GraphTypes/TransactionType.cs

```diff
@@ -11,34 +11,42 @@ public TransactionType()
         {
             Field<NonNullGraphType<TxIdType>>(
                 nameof(Transaction<T>.Id),
+                description: "A unique identifier derived from this transaction content.",
                 resolve: context => context.Source.Id
             );
             Field<NonNullGraphType<LongGraphType>>(
                 nameof(Transaction<T>.Nonce),
+                description: "The number of previous transactions committed by the signer of this transaction.",
                 resolve: context => context.Source.Nonce
             );
             Field<NonNullGraphType<PublicKeyType>>(
                 nameof(Transaction<T>.PublicKey),
+                description: "A PublicKey of the account who signed this transaction.",
                 resolve: context => context.Source.PublicKey
             );
             Field<NonNullGraphType<ByteStringType>>(
                 nameof(Transaction<T>.Signature),
+                description: "A digital signature of the content of this Transaction<T>.",
```

Comment:
Is it enough to `transaction` instead of `Transaction<T>` like other descriptions do?